src/EnergyPlus/ChillerExhaustAbsorption.cc

```diff
@@ -487,13 +500,27 @@ namespace ChillerExhaustAbsorption {
 				ShowContinueError( "Entered in " + cCurrentModuleObject + '=' + cAlphaArgs( 1 ) );
 				ShowContinueError( "resetting to WATER-COOLED, simulation continues" );
 			}
+			if ( !ExhaustAbsorber( AbsorberNum ).isEnterCondensTemp && !ExhaustAbsorber( AbsorberNum ).isWaterCooled ){
+				ExhaustAbsorber( AbsorberNum ).isEnterCondensTemp = true;
+				ShowWarningError( cCurrentModuleObject + "=\"" + cAlphaArgs( 1 ) + "\", invalid value" );
+				ShowContinueError( "Invalid to have both LeavingCondenser and AirCooled." );
+				ShowContinueError( "resetting to EnteringCondenser, simulation continues" );
+			}
 			if ( ExhaustAbsorber( AbsorberNum ).isWaterCooled ) {
-				ExhaustAbsorber( AbsorberNum ).CondReturnNodeNum = GetOnlySingleNode( cAlphaArgs( 4 ), ErrorsFound, cCurrentModuleObject, cAlphaArgs( 1 ), NodeType_Water, NodeConnectionType_Inlet, 2, ObjectIsNotParent );
-				ExhaustAbsorber( AbsorberNum ).CondSupplyNodeNum = GetOnlySingleNode( cAlphaArgs( 5 ), ErrorsFound, cCurrentModuleObject, cAlphaArgs( 1 ), NodeType_Water, NodeConnectionType_Outlet, 2, ObjectIsNotParent );
+				if ( lAlphaFieldBlanks( 4 )){
+					ShowWarningError( cCurrentModuleObject + "=\"" + cAlphaArgs( 1 ) + "\", invalid value" );
```

Comment:
And shouldn't this be testing alpha 5?  These CondReturn and CondSupply field names are confusing.